addon/components/file-upload/component.js

```diff
@@ -88,11 +88,7 @@ const component = Component.extend({
 
   attributeBindings: ['for'],
 
-  for: computed({
-    get() {
-      return `file-input-${uuid.short()}`;
-    }
-  }),
+  for: undefined,
```

Comment:
After looking at this again. Would it not be possible to have a setter for this computed prop instead of setting it in didReceiveAttrs?
Would that work?

Reply:
Sure, we can use that approach as well. Changed the code to use setter.